qtodotxt/lib/task_htmlizer.py

```diff
@@ -38,16 +38,43 @@ def task2html(self, task):
             html += ' <font color="gray">(created: {})</font>'.format(task.creation_date)
         return html
 
+    def _addUrl(self, word, color="none"):
+        if int(QtCore.QSettings().value("support_jira", 1)):
```

Comment:
Sorry, this wasn't meant to be part of the pull request, just a personal project. Guess I should have created a branch for it.